src/stress_analyzer.py

```diff
@@ -0,0 +1,157 @@
+import argparse
+from remote_execution import *
```

Comment:
Let's try to be more organized about imports.
import remote_execution as remote_exec

This just makes the code more readable. Eventually we'll move other files to reflect this.